code/modules/mob/living/carbon/xenomorph/damage_procs.dm

```diff
@@ -180,6 +180,10 @@
 		var/i = 0 //Tally up our victims.
 
 		for(var/mob/living/carbon/human/victim in range(radius,src)) //Loop through all nearby victims, including the tile.
+
+			if( (victim.wear_suit?.resistance_flags & UNACIDABLE) && (victim.head?.resistance_flags & UNACIDABLE) )
```

Comment:
Give the person acid resistance instead. And make this be checked against acid armor.

Reply:
Not sure I like this tbh. I don't want to give the armor 100% acid resist.

Reply:
The splash is of acid. No need to give it 100%, though. High enough would be fine.

Reply:
Yeah but it would be dumb for them to take acid damage. It would seriously fuck over the melee spec because of it.

Reply:
It would be inconsistent if they did take acid damage from other sources as well.

Reply:
Done.